### PR TITLE
prioritize featured tools in search results

### PR DESCRIPTION
**Pull request by g-linville**
related to gptscript-ai/desktop#116

### CODE REVIEW

File: src/lib/featured.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+export const FeaturedTools: Record<string, any> = {
```
Comment (g-linville) [inline src/lib/featured.ts]: I'm following a pattern that we use in Go a lot to maintain a set of strings (basically a `map[string]any`). Let me know if there is a better way to do this in Node.

Comment (thedadams) [reply src/lib/featured.ts]: Typescript has a Set datastructure: `let mySet = new Set<Type>();`

**Review by thedadams — APPROVED** [review]
I think using a `Set` would be cool, but, other than that, this looks good.

**Review by StrongMonkey — APPROVED** [review]

File: src/lib/featured.ts (hunk 1 above)
Comment (g-linville) [reply src/lib/featured.ts]: Updated

**Review by thedadams — APPROVED** [review]

**Pull request merged**